app/app/calculation_service/model/linear_model.py

```diff
@@ -532,6 +536,14 @@ def calculate_cluster_sigma_star(self, cluster):
             cluster_sigma_star = cluster_sigma_star * c
         return cluster_sigma_star
 
+    def calculate_total_ISU_num(self, isu_factors):
```

Comment:
can we please have a docstring explaining what this method does?
Depending on that we may be able to find a better name.